### PR TITLE
GraphQL implementation

### LINKED ISSUE #37: GraphQL

### PR DESCRIPTION
**Pull request by kermakovich**
1. Use `graphql` instead of `rest`
2. Create schema
3. Handle exception (graphql returns messages, not just internal error)
4. Remove dto, use directives in schema instead
5. Add own scalars : BigDecimal, LocalDate, Void

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -9,12 +9,12 @@
...
 	<properties>
 		<java.version>17</java.version>
-		<springboot.version>3.0.2</springboot.version>
+		<springboot.version>3.0.1</springboot.version>
 		<maven.compiler.plugin.version>3.10.1</maven.compiler.plugin.version>
 		<com.google.code.gson.version>2.10.1</com.google.code.gson.version>
 		<fasterxml.version>2.14.2</fasterxml.version>
 		<mapstruct.version>1.5.3.Final</mapstruct.version>
-		<kafka.version>1.3.16</kafka.version>
+		<kafka.version>1.3.17</kafka.version>
```
Comment (h1alexbel) [inline pom.xml]: @kermakovich should be located in the separate pull request

File: pom.xml [hunk 2]
```diff
@@ -259,6 +276,9 @@
 									<includes>
 										<include>solvd.laba.ermakovich.hu.service.**</include>
 									</includes>
+									<excludes>
+										<exclude>solvd.laba.ermakovich.hu.service.command.mapper.**</exclude>
```
Comment (h1alexbel) [inline pom.xml]: @kermakovich no exlusions here

your coverage should be clear and clean

File: src/main/java/solvd/laba/ermakovich/hu/web/config/graphql/converter/VoidConverter.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+ * @author Ermakovich Kseniya
+ */
+@Component("Void")
+public class VoidConverter implements Coercing<Void, String> {
+
+
+    @Override
+    public String serialize(final Object dataFetcherResult)
+            throws CoercingSerializeException {
+        return null;
```
Comment (h1alexbel) [inline src/main/java/solvd/laba/ermakovich/hu/web/config/graphql/converter/VoidConverter.java]: don't forget that `NULL` is terrible idea

**Review by h1alexbel — CHANGES_REQUESTED** [review]
@kermakovich check my comments, please

File: src/main/java/solvd/laba/ermakovich/hu/web/config/graphql/converter/VoidConverter.java (hunk 3 above)
Comment (kermakovich) [reply src/main/java/solvd/laba/ermakovich/hu/web/config/graphql/converter/VoidConverter.java]: I'm not sure, that `Void.TYPE.cast(new Object())`  is a good way. But it's not `NULL`

**Review by h1alexbel — APPROVED** [review]
